--- ./src/tools/include.am  	2020-01-30 16:05:09.000000000 +0100
+++ ./src/tools/include.am	2020-03-07 12:11:43.376000000 +0100
@@ -28,15 +28,7 @@
 if USE_NSS
 # ...
 else
-bin_PROGRAMS += src/tools/tor-gencert
-src_tools_tor_gencert_SOURCES = src/tools/tor-gencert.c
-src_tools_tor_gencert_LDFLAGS = @TOR_LDFLAGS_zlib@ $(TOR_LDFLAGS_CRYPTLIB)
-src_tools_tor_gencert_LDADD = \
-	$(TOR_CRYPTO_LIBS) \
-	$(TOR_UTIL_LIBS) \
-	$(rust_ldadd) \
-	@TOR_LIB_MATH@ @TOR_ZLIB_LIBS@ $(TOR_LIBS_CRYPTLIB) \
-	@TOR_LIB_WS32@ @TOR_LIB_IPHLPAPI@ @TOR_LIB_GDI@ @TOR_LIB_USERENV@ @CURVE25519_LIBS@
+# ...
 endif
 
 src_tools_tor_print_ed_signing_cert_SOURCES = src/tools/tor-print-ed-signing-cert.c
